examples/digital_fingerprinting/production/morpheus/Dockerfile

```diff
@@ -0,0 +1,83 @@
+# SPDX-FileCopyrightText: Copyright (c) 2021-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG MORPHEUS_CONTAINER=nvcr.io/nvidia/morpheus/morpheus
+ARG MORPHEUS_CONTAINER_VERSION=v22.09.00-runtime
+ARG ADDITIONAL_GROUPS
+ARG FIXUID_VERSION=0.5.1
+
+FROM ${MORPHEUS_CONTAINER}:${MORPHEUS_CONTAINER_VERSION} as common_base
+
+# Install fixuid
+RUN curl -SsL https://github.com/boxboat/fixuid/releases/download/v0.5.1/fixuid-0.5.1-linux-amd64.tar.gz | tar -C /usr/bin -xzf - && \
+    chown root:root /usr/bin/fixuid && chmod 4755 /usr/bin/fixuid && mkdir -p /etc/fixuid && \
+    bash -c 'echo -e "\
+user: morpheus\n\
+group: morpheus\n\
+paths:\n\
+  - /home/morpheus\n\
+  - /opt/conda/envs/morpheus\n\
+" | tee /etc/fixuid/config.yml >/dev/null' && \
+    \
+    # Add a non-root user
+    useradd \
+      --uid 1000 --shell /bin/bash \
+      --user-group ${ADDITIONAL_GROUPS} \
+      --create-home --home-dir /home/morpheus \
+      morpheus
+
+# Fix the entrypoint to work with different WORKDIR
+ENTRYPOINT [ "/opt/conda/bin/tini", "--", "fixuid", "-q", "/workspace/docker/entrypoint.sh" ]
+
+SHELL ["/bin/bash", "-c"]
+
+# Install vault
+RUN apt-get update \
+    && apt-get install -y \
+      apt-utils \
+      jq \
+      lsb-release \
+      software-properties-common \
+    && curl -fsSL https://apt.releases.hashicorp.com/gpg | apt-key add - \
+    && apt-add-repository "deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main" \
+    && apt-get update \
+    && apt-get install --reinstall -y  \
+      vault \
```

Comment:
Maybe remove this, unless using vault is common enough that anyone using S3 would likely need it.

Reply:
Done